src/app/policies/sources.ts

```diff
@@ -0,0 +1,44 @@
+import { DataSourceResponse } from '@/app/application/services/data-source/DataSource'
+import type KumaApi from '@/services/kuma-api/KumaApi'
+import type {
+  PolicyEntity as Policy,
+  PolicyType,
+} from '@/types/index.d'
+export interface CollectionResponse<T = Record<string, unknown>> {
+  total: number,
+  items: T[]
+  next: string | null
+}
```

Comment:
I think we might have a thing for this already, we should try and merge/reuse if we do.

Reply:
Yes, we do. Please reuse the types from `src/types/api.d.ts`.

Reply:
This is done/in-progress